src/renderer/modals/Swap/steps/StepFinished.js

```diff
@@ -110,6 +110,7 @@ const StepFinished = ({
       <Text p={20} textAlign="center" color="palette.text.shade50" ff="Inter|Regular" fontSize={4}>
         <Trans i18nKey={`swap.modal.steps.finished.description`} />
       </Text>
+      {/* FIXME: transform to Alert */}
       <Disclaimer horizontal p={2} mt={3}>
```

Comment:
I was unable to check the rendering of this component, so i just did not risk to break it.